community/modules/remote-desktop/chrome-remote-desktop/scripts/configure-grid-drivers.yml

```diff
@@ -36,6 +36,9 @@
       - gdebi-core
       - mesa-utils
       - gdm3
+      - gcc-12
+      - pkg-config
+      - libglvnd-dev
```

Comment:
Can you please add in the description how you came up with this set of packages? I am assuming we will have to touch this again and want a clear trail of breadcrumbs.

Reply:
Done.